### DISCUSSION TITLE
Function noseen in C++ ,How to find solutions?

### DISCUSSION THREAD

**Discussion by Spasov-Stefan** [community/community#69205] [Programming Help]
### Body

I kindly request your assistance in compiling this code and guidance on which functions and commands I should add to successfully achieve this. This is a part of the source code for a program I am working on, and a colleague mentioned that finding a solution for this will resolve the entire process, which is of great importance to me. I do not have much experience in compiling, so I am asking for your instructions, guidance, and assistance.

**I sincerely thank all of you in advance, as well as this website.

Best regards,**
include <iostream>

using namespace std;

void std::basic_string<>::_Rep::_M_clone(allocator *param_1,uint param_2)

{
_M_clone(param_1,param_2);
return;
}

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by satilpereira** [comment]
I would like to assist you, but to in ordre to provide the best help possible, I need a bit more context and information. Could you please provide additional details about your issue? 

But if get it right, you are having trouble compiling your code in c++?

first things first, you need  a cpp compiler instaled. you can use minGW for windows or GCC for linux, you can search online how to install any of them.

after having thigs properly installed, you can run the commands:

in a terminal
to navigate to your file:
```shell
cd /path/to/your/folder
```

and then to generate a executable version of your code you can run:
```shell
g++ -o output_file input_file.cpp
```

It will create a file that you can execute by running:
```shell
./output_file
```

Hope it helps
